docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Use Bonsai Tries to reduce database size
+sidebar_position: 12
+description: Enable this feature to reduce the size of your database
+tags:
+  - public networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Reduce database size
+
+You can use the the early access feature `--Xbonsai-limit-trie-logs-enabled` to minimize the database size of Besu, if you are using `data-storage-format=BONSAI`. When enabled, this feature can reduce database growth by more than 3GB each week on mainnet.
+
+To use this feature before it is enabled by default, add the `--Xbonsai-limit-trie-logs-enabled` option to your Besu command.
+
+## Limit Trie Logs for Bonsai
+
+:::caution
+
+The following commands are examples. Before executing these example commands on your node, m,modify them to apply to your node's configuration.
+
+:::
+
+1. Add the `--Xbonsai-limit-trie-logs-enabled` option to the [Besu configuration file](configuration-file).
+1. Stop Besu.
+1. (optional) Run the following command to run the Besu client for the Bonsai data storage format, specifying a data directory for pruned trie logs:
+   `sudo /usr/local/bin/besu/bin/besu --data-storage-format=BONSAI --data-path=/var/lib/besu --sync-mode=X_SNAP storage x-trie-log prune`
+1. Restart Besu. If you're using a `systemd` service file, as recommended by [CoinCashew](https://www.coincashew.com/coins/overview-eth/guide-or-how-to-setup-a-validator-on-eth2-mainnet/part-i-installation/step-3-installing-execution-client/besu) and [Somer](https://someresat.medium.com/guide-to-staking-on-ethereum-ubuntu-teku-f09ecd9ef2ee), ensure you execute `sudo systemctl daemon-reload`.
+1. Look for `Limit trie logs enabled: retention: 512; prune window: 30000` in your Besu configuration printout at startup.
+
+
+## Restart Besu
+
+When you restart Besu, it will begin pruning and remove unnecessary data, one block at a time. This process begins after an initial reduction in the database size during the initial startup.
```

Comment:
```suggestion
When Besu restarts, it will prune and remove unnecessary data, one block at a time. This process begins after an initial reduction in the database size during startup.
```